### ISSUE TITLE
Space should pause and resume

### ISSUE THREAD

**Issue by jlxip** [jeffvli/sonixd#439]
**Is your feature request related to a problem? Please describe.**
Pressing space should pause and resume. It's a very basic shortcut which I used a lot back when I used Spotify, but it's not here on sonixd. I often find myself pressing space either way, and some random action occurs (the one I clicked before).

**Describe the solution you'd like**
Please add a space hotkey for pausing and resuming play.

**Describe alternatives you've considered**
No alternative is needed: this is UX intuition.

**Additional context**
Thanks!

**Comment by EkaSaffronGaruda** [comment]
any update on this?

**Comment by vgambier** [comment]
I agree this is a desirable feature. For anyone looking for an alternative, if your keyboard has one, the Play/Pause Media Key works as expected.

@EkaSaffronGaruda Checkout the README, this project is in maintenance mode.
